pylint/message/message_id_store.py

```diff
@@ -128,20 +134,27 @@ def get_active_msgids(self, msgid_or_symbol: str) -> list[str]:
         # If we don't have a cached value yet we compute it
         msgid: str | None
         deletion_reason = None
+        moved_reason = None
         if msgid_or_symbol[1:].isdigit():
             # Only msgid can have a digit as second letter
             msgid = msgid_or_symbol.upper()
             symbol = self.__msgid_to_symbol.get(msgid)
             if not symbol:
                 deletion_reason = is_deleted_msgid(msgid)
+            if deletion_reason is None:
+                moved_reason = is_moved_msgid(msgid)
```

Comment:
```suggestion
                if deletion_reason is None:
                    moved_reason = is_moved_msgid(msgid)
```
Otherwise we're going to run ``is_moved_msgid`` even if ``symbol``is defined (which is all the time).